phpseclib/Math/BinaryField/Integer.php

```diff
@@ -447,40 +429,32 @@ public function negate()
 
     /**
      * Returns the modulo
-     *
-     * @return string
      */
-    public static function getModulo($instanceID)
+    public static function getModulo($instanceID): string
```

Comment:
Should be able to replace `$instanceID` with `int $instanceID`

Reply:
Done